assignments/lambda-classes.js

```diff
@@ -1 +1,181 @@
 // CODE here for your Lambda Classes
+class Person{
+    constructor(attributes){
+        this.age = attributes.age;
+        this.name = attributes.name;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+    }
+}//person
+
+class Instructor extends Person {
+    constructor(InstructorAttributes) {
+        super(InstructorAttributes)
+        this.specialty = InstructorAttributes.specialty;
+        this.favLanguage = InstructorAttributes.favLanguage;
+        this.catchPhrase = InstructorAttributes.catchPhrase;
+    }
+    // Methods
+    demo() {
+      return `Today we are learning about ${this.favLanguage} isn't that cool!`;
+    }
+    grade(student) {
+        return `${student.name} receives a perfect score on ${this.favLanguage}`
+    }
+  }// Instructor  
+   
+class ProjectManager extends Instructor { // __proto__
+    constructor(ProjectManagerAttributes) {
+      super(ProjectManagerAttributes) // this and methods
+      this.grandClassName = ProjectManagerAttributes.grandClassName;
+      this.favInstructor = ProjectManagerAttributes.favInstructor;
+      this.channel = ProjectManagerAttributes.channel;
+    }
+    // Methods
+    standUp() {
+      return `${this.name} announces to ${this.channel}, @channel standy times!`;
+    }
+    debugsCode(student,instructor) {
+        return `${this.name} debugs ${student.name}'s code on ${instructor.favLanguage}`;
+    }
+  }// ProjectManager
+  
+  class Student extends ProjectManager { // __proto__
```

Comment:
we dont want a student to have all the functions and attributes of a project manager, this should extend Person
```suggestion
  class Student extends Person{ // __proto__
```
try and get in the mindset of thinking about these proto parent child relationships more of as a tree or a water fall, rather then a a linear path.
and think about what you your code is doing, what functions and properties do we want the student objects to have?
we certainly don't want then to be able to grade or lead standups, that's the instructor and PM job.